bind/gengo.go

```diff
@@ -111,6 +114,28 @@ func cgopy_decref(ptr unsafe.Pointer) {
 	refs.Unlock()
 }
 
+//export cgo_check
+func cgo_check() {
+	godebug := os.Getenv("GODEBUG")
+	cgocheck := -1
+	var err error
+	if godebug != "" {
+                for _, option := range strings.Split(godebug, ",") {
+                        if strings.HasPrefix(option, "cgocheck=") {
```

Comment:
instead of checking if it has the correct prefix, check whether it isn't there and `continue`.
this decreases the amount of indentation.